### PR TITLE
ban public users config option v0

### PR DESCRIPTION
**Pull request by meffij**
could not run unit tests locally, Dima please review

### CODE REVIEW

**Comment by meffij** [comment]
Would also like to add tests eventually, need guidance on that

**Comment by akarve** [comment]
@meffij the act of submitting the PR will run the unit tests; looks like travis/pr failed

File: registry/quilt_server/config.py [hunk 1]
```diff
@@ -6,6 +6,13 @@
 
 See `app.config.from_object('...')` in __init__.py.
 """
+import os
+
+BAN_PUBLIC_USERS = os.getenv('QUILT_BAN_PUBLIC_USERS', '')
+if len(BAN_PUBLIC_USERS):
```
Comment (dimaryaz) [inline registry/quilt_server/config.py]: `if BAN_PUBLIC_USERS`

**Comment by dimaryaz** [comment]
Can you change the `api` decorator instead of updating each call to it?

File: registry/quilt_server/views.py [hunk 2]
```diff
@@ -31,6 +31,7 @@
 from .models import (Access, Customer, Instance, Invitation, Log, Package,
                      S3Blob, Tag, UTF8_GENERAL_CI, Version)
 from .schemas import LOG_SCHEMA, PACKAGE_SCHEMA
+from .config import BAN_PUBLIC_USERS
```
Comment (dimaryaz) [inline registry/quilt_server/views.py]: Sorry, one more thing... You should use `app.config` instead of importing it directly. See the code below.

**Pull request merged**